src/tests/Interop/PInvoke/Generics/GenericsTest.VectorU.cs

```diff
@@ -69,7 +69,12 @@ public unsafe partial class GenericsTest
     [ActiveIssue("https://github.com/dotnet/runtimelab/issues/177", typeof(TestLibrary.Utilities), nameof(TestLibrary.Utilities.IsNativeAot))]
     public static void TestVectorU()
     {
-        if (Vector<uint>.Count == 8)
+        if (Vector<uint>.Count == 16)
+        {
+            // TODO: P/Invoke tests do not yet handle 512-bit Vector<T>
```

Comment:
This needs to either be handled or have a tracking issue that is picked up shortly after merge in a follow up PR.

Reply:
I worded that comment oddly.  The issue is not specific to `Vector<T>` but rather `Vector512` in general isn't covered by these interop tests.  Do we have something tracking that already?

Reply:
Don't believe so. These tests are really centered around Vector<T> and most of the V128/256/512 tests are elsewhere and newer/different

Reply:
These are under [tests/Interop/PInvoke/Generics](https://github.com/dotnet/runtime/tree/main/src/tests/Interop/PInvoke/Generics), which AFAICT cover scenarios involving all the generic types that have specific ABI handling.  What I mean is that we don't currently have any tests for `Vector512` ABI handling.  Do we even have support for that?

Reply:
> Do we even have support for that?

Yesn't. It's expected to throw today, but the tests are still setup since there's only a single known issue actually blocking the support (which is that the default calling convention for vectors on Windows should be in register, not via a return buffer).

Reply:
> the tests are still setup

This is the part I'm confused about. `Vector512` is conspicuously absent from the list of types covered by the tests under https://github.com/dotnet/runtime/tree/main/src/tests/Interop/PInvoke/Generics.

I'm happy to open a tracking issue for adding them, but I would have assumed that would be an implicit part of enabling the ABI support.

I'm now concerned we may be missing some coverage, though. If `Vector512` used in P/Invoke is supposed to throw, we'd need to be sure the same happens for 512-bit `Vector<T>`, but I don't see where we're testing either.